src/main/java/nl/bioinf/minorapplicationdesign/ontpillen/model/web_scraping/DrugFetcher.java

```diff
@@ -59,29 +59,34 @@ private void storeDrugsInDao(List<Element> drugGroups){
         }
 
         currentDrug = (DrugGroup) drugDao.getDrugByName(currentDrugElement.text());
-        LOGGER.debug("Storing: " + currentDrug.getName() + " in the dao");
 
+        addDrugs(drugGroups, currentDrugElement, currentDrug);
+        drugGroups.remove(0);
+
+//        As long as there are elements in this.drugGroups run this method again
```

Comment:
Javadoc klopt niet meer door extracten van methode, ik vraag me af of het handig is dit te extracten omdat het hierdoor misschien minder duidelijk is dat de methode recursief is.